portal-api/src/v1/api.js

```diff
@@ -269,6 +269,29 @@ const findLatestGefMandatoryCriteria = async () => {
   }
 };
 
+/**
+ * Validates utilisation report data returning any errors to display to the user
+ * @param {import('@ukef/dtfs2-common').UtilisationReportCsvRowData[]} reportData
+ * @returns {Promise<import('./api-response-types').ValidateUtilisationReportDataResponseBody>} Object containing the validation errors to display to the user
```

Comment:
I'm not sure I follow what you are suggesting? I have created a type for this response that we refer to here to try and make this as simple as possible.

Reply:
The import documentation seems overkill to me, I think something like this would be a lot simpler and easier to read/maintain:

```ts
* @param {UtilisationReportCsvRowData} reportData
 * @returns {Promise<ValidateUtilisationReportDataResponseBody>}
```

Reply:
So the issue is if I just type it like that then you can't click through to the type definition, and also we don't get type safety in the following file as it has no idea what that type is.

If I do your change and add `const { UtilisationReportResponseBody } = require('./api-response-types');` at the top of the api.js file, then I can click through to the definition, and we do get the type safety in the files calling the function but I get a linting error violating the `no-unused-vars` rule since this is only used in the docstring.

Let me know if you have a solution!